### PR TITLE
Round 2 - updates to 800-171 profile

### PR DESCRIPTION
**Pull request by shawndwells**
** do not merge yet **

### CODE REVIEW

**Comment by shawndwells** [comment]
OK to merge once/if jenkins passes :)

**Comment by shawndwells** [comment]
s/merge/review for merge/r

File: RHEL/7/input/profiles/nist-800-171-cui.xml [hunk 1]
```diff
@@ -1,19 +1,16 @@
 <Profile id="nist-800-171-cui">
-<title>RHEL7 Server Baseline for Controlled Unclassified Information in Nonfederal Information Systems and Organizations (NIST 800-171)</title>
-<description override="true">
-
-From NIST 800-171, Section 2.2:
+<title>Unclassified Information in Nonfederal Information Systems and Organizations (NIST 800-171)</title>
```
Comment (redhatrises) [inline RHEL/7/input/profiles/nist-800-171-cui.xml]: s/Nonfederal/Non-Federal/g

File: shared/xccdf/services/ssh.xml [hunk 2]
```diff
@@ -715,10 +715,9 @@ functions. The only SSHv2 hash algorithms meeting this requirement is SHA2.
 <ident prodtype="rhel7" cce="27455-5" />
 <oval id="sshd_use_approved_macs" />
 <ref prodtype="rhel7" nist="AC-17(2),IA-7,SC-13" disa="1453"
-ossrg="SRG-OS-000250-GPOS-00093" stigid="040620" cui="3.1.13" />
+ossrg="SRG-OS-000250-GPOS-00093" stigid="040620" cui="3.1.13, 3.13.11" />
```
Comment (redhatrises) [inline shared/xccdf/services/ssh.xml]: Slight nitpick, extra whitespace between 3.1.13 and 3.13.11

File: shared/xccdf/system/software/integrity.xml [hunk 3]
```diff
@@ -241,8 +241,8 @@ have not been altered. These hashes must be FIPS 140-2 approved cryptographic ha
 </rationale>
 <ident prodtype="rhel7" cce="80377-5" />
 <oval id="aide_use_fips_hashes"/>
-<ref prodtype="rhel7" nist="SI-7.1" disa="366"
-ossrg="SRG-OS-000480-GPOS-00227" stigid="021620" />
+<ref prodtype="rhel7" nist="SI-7(1)" disa="366" ossrg="SRG-OS-000480-GPOS-00227"
```
Comment (redhatrises) [inline shared/xccdf/system/software/integrity.xml]: Slight nitpick: end of line whitespace.

**Comment by redhatrises** [comment]
@shawndwells can you rebase and fix the nitpicks?

**Comment by shawndwells** [comment]
rebased & fixed incremental patches

**Comment by redhatrises** [comment]
[retest this please]

**Comment by redhatrises** [comment]
Ack. Manually removed a duplicate entry in the CUI profile and merged.

**Pull request merged**